stylesheets/_mixins.scss

```diff
@@ -1,3 +1,13 @@
+@mixin remove-button-styles() {
```

Comment:
Sometimes this is called `normalize-*` or `reset-*`:

- https://cssreset.com/what-is-a-css-reset/
- https://necolas.github.io/normalize.css/